definitions/features/candlepin.rb

```diff
@@ -7,6 +7,22 @@ class Features::Candlepin < ForemanMaintain::Feature
     end
   end
 
+  def current_version
+    @current_version ||= rpm_version(package_name)
+  end
+
+  def current_minor_version
```

Comment:
Same for the `current_minor_version` and `package_name` method. It can also be removed.